test/configs/tls_noip_b.conf

```diff
@@ -0,0 +1,21 @@
+# TLS config file
+# Cert has no IPs, so we use an IP here to simulate a single hostname cluster.
```

Comment:
Travis also has support to set custom hostnames which may help to simulate setup further https://docs.travis-ci.com/user/hosts/

Reply:
Nice I did not know that! Would prefer a test that can be run locally as well. What is best way to have a test understand if it is running on Travis?

Reply:
I think we could check whether `os.Getenv("TRAVIS") == "true"` which is one of the default environment variables and skip locally if not defined for example.

Reply:
Good point. For now I think we are ok, we are using cert with localhost and no IPs defined, so is testing what we want.